templates/.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "Feature Request: ..."
+labels: enhancement
+assignees: 
+---
+
+## what
+* Describe *what* changes you would like made to this project. 
```

Comment:
```suggestion
* Describe *what* changes you would like to make to this project. 
```

Reply:
Doesn't that imply that the user would like to make them? Versus some else making them...

Reply:
maybe 
```
Describe *what* changes you would like to be made to this project.
```